### DISCUSSION TITLE
How to cancel a spellcast then spell already picked on Android w/o keyboard?

### DISCUSSION THREAD

**Discussion by roman-yagodin** [ihhub/fheroes2#6938] [Q&A] [answered]
So the question. Using keyboard, I'd just press ESC, but how to do this using touch interface?..

**Comment by Mr-Bajs** [comment, marked as answer]
Do a 2-finger touch on the battlefield. It cancels it

**Reply by oleg-derevenetz** [reply]
That's right. To be more accurate: you can cancel the spellcast either by pressing Esc key or by pressing the right mouse button. On touchscreens, there is no Esc key, but right mouse button is emulated by two-finger touch event, so you can use the two-finger touch to cancel the spellcast.

**Comment by roman-yagodin** [comment]
Thanks! 👍
